site/_docs/continuous-integration.md

```diff
@@ -118,6 +118,7 @@ customizable. If your script won't change much, you can write your test
 incantation here directly:
 
 {% highlight yaml %}
+before_install: gem install jekyll html-proofer
```

Comment:
True, but do we really want that lack of documentation holding up getting this documentation in place? This provides a tangible benefit for people who want to get their site up and running with CI somewhere, but without having to know Ruby bits.